locust/web.py

```diff
@@ -477,4 +514,59 @@ def update_template_args(self):
             "stats_history_enabled": options and options.stats_history_enabled,
             "tasks": dumps({}),
             "extra_options": extra_options,
+            "show_locustfiles": self.show_locustfiles,
         }
+
+    def update_environment(self, locustfile, user_classes, shape_class, events, parsed_options):
+        options = parsed_options
```

Comment:
I’m not a huge fan of renaming variables. Probably best to just remove these three lines and refer to everything by their full name.